docs/ref/templates/builtins.txt

```diff
@@ -1054,8 +1054,11 @@ this example, the space around ``Hello`` won't be stripped::
 
 Outputs one of the syntax characters used to compose template tags.
 
-Since the template system has no concept of "escaping", to display one of the
-bits used in template tags, you must use the ``{% templatetag %}`` tag.
+The template system has no concept of “escaping” individual characters.
+However, you can use the {% templatetag %} tag to display one of the bits used
```

Comment:
You need to restore the backticks around the `templatetag` part here. Also, I would use "characters" rather than "bits," which would be more straightforward -- something like, "to display one of the template tag character combinations."